package-system/openimageio-opencolorio/test/.gitignore

```diff
@@ -0,0 +1,4 @@
+# test image files generated by our tests:
+temp_save_image*
+
+
```

Comment:
Nit: This wouldn't be necessary if the test didn't leave artifacts around.

Reply:
I'll fix these.  And amissing carriage return.  it doesn't seem like I need to publish a new package for these fixes, at least.